Aztec/Classes/Public/TextKit/TextAttachment.swift

```diff
@@ -106,7 +123,13 @@ public class TextAttachment: NSTextAttachment
     /// Otherwise, we'll always take the whole container's width.
     ///
     override public func attachmentBoundsForTextContainer(textContainer: NSTextContainer?, proposedLineFragment lineFrag: CGRect, glyphPosition position: CGPoint, characterIndex charIndex: Int) -> CGRect {
-
+        self.textContainer = textContainer
+        switch kind {
+        case .RemoteImage:
+            startImageFetching()
+        default:
+            print("No action to do")
```

Comment:
No need to log this (as there's not much value in the log message).  Add a comment if we need to document it's ok if nothing happens.